source/class/cv/io/timeseries/FluxSource.js

```diff
@@ -167,6 +167,9 @@ qx.Class.define('cv.io.timeseries.FluxSource', {
         lineEntries = line.split(',');
         if (lineEntries[valueIndex]) {
           value = parseFloat(lineEntries[valueIndex]);
+          if (isNaN(value)) {
+            value = 0;
+          }
```

Comment:
Is `NaN` really equal to `0` - or should this value be skipped as (e.g.) it doesn't exist?

Reply:
Unfortunately I had this fix in my local system for a long time, so I do not remember how to reproduce the problem, all I remember that this causes an error. So I do not have an answer if setting to 0 or leaving it out is the better option. Usually chart do not handle gaps well, or you don't event see that there is a gap/problem because the line from the former to the next value just goes over it. So for now I would go with the 0 because you can see the spikes in the chart (although its technically the wrong value)